src/Input.test.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+describe('Input Component', () => {
+  const handleChange = jest.fn();
+  const handleClick = jest.fn();
+
+  const renderInput = () => render((<Input
+    value="과제하기"
+    onChange={handleChange}
+    onClick={handleClick}
+  />));
```

Comment:
```suggestion
    />
  ));
```

줄바꿈해주시면 더 자연스러울 것 같아요.